src/legacy/server/logging/configuration.js

```diff
@@ -64,6 +64,7 @@ export default function loggingConfiguration(config) {
       json: config.get('logging.json'),
       dest: config.get('logging.dest'),
       timezone: config.get('logging.timezone'),
+      streamErrorHandling: config.get('logging.streamErrorHandling'),
```

Comment:
NIT: totally cool as is.

do we want to make this even further?

like it accept a array of strings? and if the array.includes(error.code) that it can bypass it?

if the only plan for ENOSPC, then probably can switch the name to something like`logging.ignoreEnospcError` or something no space.

Reply:
good call. I modified the feature flag name to ignoreEnospcError